### PR TITLE
vtu ouput update

### PR DESCRIPTION
**Pull request by JohnVirtonomy**

### CODE REVIEW

File: SPHINXsys/src/shared/bodies/base_body.cpp [hunk 1]
```diff
@@ -184,7 +190,7 @@ namespace SPH
 			if (fabs(phi) < particle_spacing_min_)
 				tagAParticle(i);
 		}
-		std::cout << "Number of surface particles : " << body_part_particles_.size() << std::endl;
+		//std::cout << "Number of surface particles : " << body_part_particles_.size() << std::endl;
```
Comment (BenceVirtonomy) [inline SPHINXsys/src/shared/bodies/base_body.cpp:193]: why did you change this?

**Review by BenceVirtonomy — APPROVED** [review]

File: SPHINXsys/src/shared/bodies/base_body.cpp (hunk 1 above)
Comment (JohnVirtonomy) [reply SPHINXsys/src/shared/bodies/base_body.cpp:193]: When the surface particles were called each time, it was put out each time step
